learn/getting_started/installation.mdx

```diff
@@ -148,3 +150,27 @@ To deploy Meilisearch on a cloud service, follow one of our dedicated guides:
 - [Koyeb](/learn/cookbooks/koyeb.md)
 - [Qovery](/learn/cookbooks/qovery.md)
 - [Railway](/learn/cookbooks/railway.md)
+
+## Supported operating systems
+
+This section lists operating systems Meilisearch officially supports and tests with every new release. Meilisearch binaries might still run in unsupported environments. Refer to the [changelog](https://github.com/meilisearch/MeiliSearch/releases) for more information on operating system support changes.
```

Comment:
To me it does not seem helpful to point people to the changelog for information about operating system support, as any relevant info will be buried among a lot of other stuff. This page exists to collect that information so I don't see the value in sending people elsewhere.

```suggestion
This section lists operating systems Meilisearch officially supports and tests with every new release. Meilisearch binaries might still run in unsupported environments. 
```